nano/secure/ledger.cpp

```diff
@@ -183,6 +183,19 @@ void ledger_processor::state_block (nano::state_block const & block_a)
 			{
 				prev_balance = ledger.balance (transaction, block_a.hashables.previous);
 			}
+			else if (result.verified == nano::signature_verification::unknown)
+			{
+				// Check for possible regular state blocks with epoch link (send subtype)
+				result.verified = validate_message (block_a.hashables.account, block_a.hash (), block_a.signature) ? nano::signature_verification::unknown : nano::signature_verification::valid;
```

Comment:
This would be more readable as a `if` statement rather than a ternary and I think we should optimize for readability here

```suggestion
				if (validate_message (block_a.hashables.account, block_a.hash (), block_a.signature))
				{
					result.verified = nano::signature_verification::unknown
				}
				else
				{
					result.verified = nano::signature_verification::valid;
				}
```